src/jedi_bundle/utils/git.py

```diff
@@ -66,29 +66,40 @@ def repo_is_reachable(logger, url, username, token):
 # --------------------------------------------------------------------------------------------------
 
 
-def repo_has_branch(logger, url, branch, is_tag=False):
+def repo_has_branch(logger, url, branch, is_tag=False, is_commit=False):
```

Comment:
You're presenting on type annotations tomorrow! Practice what you preach 😉 

```suggestion
def repo_has_branch(
    logger: Logger,
    url: str,
    branch: str,
    is_tag: bool = False,
    is_commit: bool = False
) -> bool:
```

NOTE: You'll have to import `Logger` at the top of this file for this annotation to work.

Reply:
Hey now, I'm just presenting on it. I'm not preaching it yet lol